### DISCUSSION TITLE
如何修改控制台密码?

### DISCUSSION THREAD

**Discussion by wangshan9870** [alibaba/higress#484] [General]
如何修改控制台密码?

**Comment by CH3CHO** [comment]
UI 上暂时还不支持修改，不过可以执行下面的命令进行操作：

```bash
kubectl edit secret higress-console -n higress-system
```

修改 `adminPassword` 的值即可。如果原位修改的话，是需要填入 Base64 编码后的值的。

**Comment by CH3CHO** [comment]
最新版本已经支持修改管理员密码了

![image](https://github.com/alibaba/higress/assets/2909796/b5633591-5682-4f6f-9ac2-53ab0b4ac7b0)

**Reply by ukco** [reply]
我是否可以通过预设的方式，将用户名和密码存放到配置文件中？这样每次我重新编译打包构建，发布部署的时候就能避免，首次登录控制台就要进行初始化设置。

---
Can I store the username and password in the configuration file in a preset way? In this way, every time I recompile and package the build, it can be avoided when I publish the deployment, and the initialization setting is required for the first time when I log in to the console.

**Reply by CH3CHO** [reply]
是可以的。而且用 helm 安装也可以通过 `--set` 参数来制定用户名密码：`--set higress-console.admin.username=xxx --set higress-console.admin.displayName=xxx --set higress-console.admin.password=xxx`

---
It's OK. Moreover, using helm to install, you can also use the `--set` parameter to set the user name and password: `--set higress-console.admin.username=xxx --set higress-console.admin.displayName=xxx --set higress-console.admin.password=xxx`
